src/main/java/org/motechproject/newebodac/mapper/CampaignMessageMapper.java

```diff
@@ -20,4 +23,23 @@
   @Override
   @Mapping(target = "id", ignore = true)
   CampaignMessage fromDto(CampaignMessageDto campaignMessageDto);
```

Comment:
add mapping for visitTypeId here too, also ignore create and update dates

Reply:
fixed